airflow/ui/src/components/TriggerDag/TriggerDAGForm.tsx

```diff
@@ -131,15 +129,8 @@ const TriggerDAGForm = ({ dagId, onClose, open }: TriggerDAGFormProps) => {
 
   return (
     <>
-      <Accordion.Root
-        collapsible
-        defaultValue={[flexibleFormDefaultSection]}
```

Comment:
I needed this to have the first section of accordion expanded if provided. ELse all sections are folded and you need a click to start changing params